src/commands/test.ts

```diff
@@ -387,11 +388,26 @@ async function processTestScripts(
         }
 
         console.log(`Run test function ${test}`);
-        await testSuiteInstance[test]();
-        if (expectedRevertMessage !== undefined) {
-          throw new Error(
-            `Test was expected to revert with message "${expectedRevertMessage}" but no revert happened.`
-          );
+
+        const noOfFuzzingParameters = testSuiteInstance[test].length;
```

Comment:
Maybe rename to `numOfFuzzingParameters` because 'no' somehow hints at a boolean

Reply:
Changed